api/discord.js

```diff
@@ -0,0 +1,19 @@
+import { Client, Intents } from 'discord.js';
+
+export class Discord {
+    static instance = null;
+
+    static getReadyInstance = async() => {
+        if (Discord.instance !== null) {
+            return discord;
+        }
+        console.log('getting new instance');
+        
+        const client = new Client({ intents: [Intents.FLAGS.GUILDS] });
+        
+        // wait for the client to be ready
+        await client.login()
```

Comment:
This worked for daily updates, not sure if we really need to call client is ready, seems like the promise only returns when the client is ready.